app/assets/javascripts/application.js

```diff
@@ -0,0 +1,3 @@
+//= require govuk_publishing_components/all_components
+//= require chartkick
```

Comment:
If only the chart component uses chartkick (and there are no plans for other components to use it, would be good to move the require in `chart.js`